docs/proposals/PROPOSAL_I.md

```diff
@@ -0,0 +1,235 @@
+# Proposal I - compatibilities extention in platform object
+
+This proposal is born from proposal A to H.
+
+The purpose of the proposal, based on the maintainers feedback, is to address the concern maintaining standalone artifact in OCI.
+
+## image spec platform object extention
+
+### revive platform.features
+
+platform.features will be reused for image selection, refer to [proposal E](https://github.com/opencontainers/wg-image-compatibility/blob/main/docs/proposals/PROPOSAL_E.md)
+
+### add platform.compatibilities descriptor
+
+Add a new descriptor in platform object in [image index](https://github.com/opencontainers/image-spec/blob/main/image-index.md#image-index-property-descriptions).
+
+- **`manifests`** _array of objects_
+
+  - **`platform`** _object_
+
+    This OPTIONAL property describes the minimum runtime requirements of the image.
+    This property SHOULD be present if its target is platform-specific.
+
+    - **`architecture`** _string_
+
+    - **`variant`** _string_
+
+    - **`os`** _string_
+
+    - **`os.version`** _string_
+
+    - **`os.features`** _array of strings_
+
+    - **`features`** _array of strings_
+
+    - **`compatibilities`** _[descriptor](https://github.com/opencontainers/image-spec/blob/main/descriptor.md)_
+
+      This OPTIONAL property specifies a descriptor of a compatibilities manifest.
+
+## Compatibilities Manifest
+
+The compatibilities manifest is a special image manifest, contains only config and no blob layer.
+
+```json
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "config": {
+    "mediaType": "application/vnd.oci.image.config.v1+json",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 678
+  },
+  "layers": [],
+  "annotations": {
+    "oci.opencontainers.image.created": "2024-03-27T08:08:08Z"
+  }
+}
+```
+
+### Config Extention
```

Comment:
Instead of extending the config, I think, it's more reasonable to add a new schema for new image compatibility media type under image spec.
https://github.com/opencontainers/image-spec/blob/main/media-types.md#oci-image-media-types

Reply:
it could be another option. will update in next commit

Reply:
We should not bring image config to the index level. The descriptor idea makes only sense to me when we create a new compatibility media type. I think this proposal and J should be combined to one.

Reply:
>  I think this proposal and J should be combined to one.

+1 - it would be easier to have one WIP proposal.

Reply:
Just explained to Marcin, it's two different proposals. I will update the api call path in the proposal how to get the compatibilities

Reply:
some stuff to help understand proposal I and proposal J: https://docs.google.com/presentation/d/1w_MNe2qEJH6Izcq61IABPfx_C9cUY1405j_FgCs8e1Q/edit?usp=sharing

Reply:
Yup, indeed those are two different proposals, although as I mentioned, the proposal violates the existing media types relations: https://github.com/opencontainers/image-spec/blob/main/media-types.md#relations

Reply:
Closing this thread as there is a new proposal.